### ISSUE TITLE
Unable to parse version "25.1-jre-graal-sub-1"

### ISSUE THREAD

**Issue by kmate-ct** [kevin-lee/just-semver#216]
# Bug

The library gives
```
InvalidVersionStringError(25.1-jre-graal-sub-1)
```
for the above version

# Summary

# Project Details
Version: 0.13.0
Scala Version: 2.12.18
Java Version: openjdk11

# Description
**Describe the bug**

I expect that version to parse.

**To Reproduce**

```scala
SemVer.parse("25.1-jre-graal-sub-1")
```

**Expected behavior**

The version parses without error.

**Comment by kmate-ct** [comment]
Well, never mind if this doesn't fit the scope of the lib. There are many wild libraries like this that doesn't really use SemVer, but something close to it. There are also things where there are 4 dot-separated digits... it'd be just great to have a library that compares these versions correctly off the shelf.

**Comment by kevin-lee** [comment]
@kmate-ct Sorry I just saw this issue. Well, as you mentioned, it's out of the scope of this library as `25.1-jre-graal-sub-1` is not a valid SemVer.
However, I agree that it would be good to support other versions. Let me see what I can do.

**Comment by kevin-lee** [comment]
This is solved by the new `DecVer` (#230 / PR $231). It's from the new module `just-semver-decver` and it will be released soon.
